.github/workflows/docs.yml

```diff
@@ -1,20 +1,12 @@
 name: Documentation
 
 on:
+  push:
+    branches:
+      - main
   release:
     types:
       - published
-  workflow_dispatch:
```

Comment:
I have the feeling that disallowing any manual trigger of this wont benefit us

Reply:
this makes sense to me

Reply:
Included and tested on my fork, hence resolving.